### PR TITLE
PP-1500 Added missing integration tests for EPDQ notification endpoint

### PR DESCRIPTION
**Pull request by ckalista**
## WHAT
_A brief description of the pull request:_

A new endpoint for handling EPDQ notifications has been introduced with an oversight to provide proper integration test coverage for it. `EpdqNotificationResourceITest` rectifies this.

### CODE REVIEW

**Comment by ckalista** [comment]
rebuild

File: src/test/java/uk/gov/pay/connector/it/resources/epdq/EpdqNotificationResourceITest.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+import static javax.ws.rs.core.MediaType.APPLICATION_FORM_URLENCODED;
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static uk.gov.pay.connector.model.domain.ChargeStatus.*;
+import static uk.gov.pay.connector.model.domain.GatewayAccount.CREDENTIALS_SHA_OUT_PASSPHRASE;
+
+public class EpdqNotificationResourceITest extends ChargingITestBase {
+
+    private static final String RESPONSE_EXPECTED_BY_WORLDPAY = "[OK]";
```
Comment (alexbishop1) [inline src/test/java/uk/gov/pay/connector/it/resources/epdq/EpdqNotificationResourceITest.java]: I don’t think so!

File: src/test/java/uk/gov/pay/connector/it/resources/epdq/EpdqNotificationResourceITest.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    private static final String NOTIFICATION_PATH = "/v1/api/notifications/epdq";
+
+    public EpdqNotificationResourceITest() {
+        super("epdq");
+    }
+
+    @Test
+    public void shouldHandleAWorldpayNotification() throws Exception {
```
Comment (alexbishop1) [inline src/test/java/uk/gov/pay/connector/it/resources/epdq/EpdqNotificationResourceITest.java]: Still not Worldpay!

**Review by alexbishop1 — CHANGES_REQUESTED** [review]
Looks good but a few references to Worldpay slipped through!

I think it would be valuable to add a test to check nothing happens to the payment if the SHASIGN isn’t correct.

File: src/test/java/uk/gov/pay/connector/it/resources/epdq/EpdqNotificationResourceITest.java (hunk 1 above)
Comment (ckalista) [reply src/test/java/uk/gov/pay/connector/it/resources/epdq/EpdqNotificationResourceITest.java]: Indeed and addressed.

File: src/test/java/uk/gov/pay/connector/it/resources/epdq/EpdqNotificationResourceITest.java (hunk 2 above)
Comment (ckalista) [reply src/test/java/uk/gov/pay/connector/it/resources/epdq/EpdqNotificationResourceITest.java]: Addressed.

**Comment by ckalista** [comment]
All code review suggestions have been applied.

File: src/test/java/uk/gov/pay/connector/it/resources/epdq/EpdqNotificationResourceITest.java [hunk 3]
```diff
@@ -24,24 +24,24 @@
 
 public class EpdqNotificationResourceITest extends ChargingITestBase {
 
-    private static final String RESPONSE_EXPECTED_BY_WORLDPAY = "[OK]";
+    private static final String RESPONSE_EXPECTED_BY_EPDQ = "[OK]";
```
Comment (alexbishop1) [inline src/test/java/uk/gov/pay/connector/it/resources/epdq/EpdqNotificationResourceITest.java]: I don’t think ePDQ actually care, so maybe we shouldn’t assert this

**Review by alexbishop1 — DISMISSED** [review]

**Review by alexbishop1 — APPROVED** [review]

**Pull request merged**